airflow/www/views.py

```diff
@@ -963,14 +969,28 @@ def task_stats(self, session=None):
         else:
             final_task_instance_query_result = running_task_instance_query_result.subquery('final_ti')
 
-        qry = session.query(
-            final_task_instance_query_result.c.dag_id,
-            final_task_instance_query_result.c.state,
-            sqla.func.count(),
-        ).group_by(final_task_instance_query_result.c.dag_id, final_task_instance_query_result.c.state)
+        qry = (
+            session.query(
+                final_task_instance_query_result.c.dag_id,
+                final_task_instance_query_result.c.state,
+                final_task_instance_query_result.c.is_dag_running,
+                sqla.func.count(),
+            )
+            .group_by(
+                final_task_instance_query_result.c.dag_id,
+                final_task_instance_query_result.c.state,
+                final_task_instance_query_result.c.is_dag_running,
+            )
+            .order_by(final_task_instance_query_result.c.is_dag_running.desc())
+        )
 
         data = {}
-        for dag_id, state, count in qry:
+        has_running_dags = False
+        for dag_id, state, is_dag_running, count in qry:
+            if is_dag_running:
+                has_running_dags = True
+            elif has_running_dags:
+                break
```

Comment:
What is `has_running_dags` supposed to  do? Because right now with `break`, if any dag is running then all other dags won't even show any statuses.

<img width="625" alt="Screen Shot 2022-02-09 at 5 01 34 PM" src="https://user-images.githubusercontent.com/4600967/153297991-8f5393f5-6c12-4255-aea5-c11a7733e1e7.png">